src/app/shell/personal-cabinet/provider/provider-org-info/provider-org-info.component.html

```diff
@@ -1,8 +1,11 @@
-<div *ngIf="provider$ | async as provider" class="container">
-  <app-provider-info
-    [provider]="provider"
-    [currentStatus]="currentStatus"
-    (selectedTabChange)="onTabChanged($event)"
-    [isProviderView]="true"
-  ></app-provider-info>
-</div>
+<ng-container *ngIf="provider$ | async as provider">
+  <app-status-banner [editLink]="'/create-provider/info'" [providerStatus]="providerStatus"></app-status-banner>
+  <div class="container">
+    <app-provider-info
+      [provider]="provider"
+      [currentStatus]="currentStatus"
+      (selectedTabChange)="onTabChanged($event)"
+      [isProviderView]="true"
```

Comment:
events should be listed as the last


```suggestion
[provider]="provider"
[isProviderView]="true"
(selectedTabChange)="onTabChanged($event)"
```
[currentStatus]="currentStatus"
(selectedTabChange)="onTabChanged($event)"
[isProviderView]="true"